src/App.tsx

```diff
@@ -1,5 +1,49 @@
+import colors from './colors.json';
+
+function ColorObject({ colorInfo }) {
```

Comment:
Use more meaningful names for components. Otherwise, everything will be a `____Object`. Ex. this could be ColorText or ColorName.